4-tools/presentation.ipynb

```diff
@@ -765,6 +779,21 @@
     "    sleep(0.5)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Git\n",
+    "\n",
+    "Git is a free and open-source distributed version control system designed to handle everything from small to very large projects with speed and efficiency.\n",
```

Comment:
Git is designed and very good at handling source code. Git is not designed nor very good at handling large binary files, although extensions like [git-lfs](https://git-lfs.com/) exist.